ts/src/flexible-event/privacy.test.ts

```diff
@@ -74,20 +74,43 @@ const infoGainTests = [
   {
     numStates: 3,
     epsilon: 14,
-    expected: 1.584926511508231,
+    expected: 1.5849265115082316,
   },
   {
     numStates: 1,
     epsilon: 14,
     expected: 0,
   },
+  {
+    numStates: 2925,
+    epsilon: 14,
+    attributionScopeLimit: 3,
+    expected: 11.464610112285094,
+  },
+  {
+    numStates: 2925,
+    epsilon: 14,
+    attributionScopeLimit: 5,
+    expected: 11.467486215356272,
+  },
+  {
+    numStates: 2925,
+    epsilon: 14,
+    attributionScopeLimit: 100,
+    expected: 11.597577197906126,
+  },
 ]
 
 void test('maxInformationGain', async (t) => {
   await Promise.all(
     infoGainTests.map((tc, i) =>
       t.test(`${i}`, () => {
-        const actual = maxInformationGain(tc.numStates, tc.epsilon)
+        const actual = maxInformationGain(
+          tc.numStates,
+          tc.epsilon,
+          tc.attributionScopeLimit ?? 1,
+          constants.defaultMaxEventStates
```

Comment:
Please add a test case that does not use `defaultMaxEventStates`.

Reply:
Done.